cinema/views.py

```diff
@@ -35,7 +36,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    authentication_classes = (JWTAuthentication,)
```

Comment:
Is it necessary to state JWTAuthentication here (and in all other views)? In settings you set it as default for the whole project

Reply:
It doesn't work without authentication_classes = (JWTAuthentication,) I checked everything twice

Reply:
@Nattalli heeellppppp please!!!